### DISCUSSION TITLE
I need a recommendation: programatically scrapping the results of a previous job

### DISCUSSION THREAD

**Discussion by DirgoSalga** [jaypyles/Scraperr#52] [Q&A]
I'm trying to achieve the following:

There is a site I am scraping that lists a variety of videos. From the main page, I can obtain the titles and source URLs for the images. If I then click on the link to the video itself, I can scrape the description from that page.

Is there currently a way to do this? I could write my own Python script to scrape through the results of a job I ran previously, but that somewhat defeats the purpose. How would you use Scraperr to solve this problem?

Thank you!

**Comment by Alyxfranck** [comment]
In your case iam not sure how it could work with videos. I would require a lot of storage.. 

I have myself wrote a Python Script that gets the data from jobs with a large list of ursl and saves them to json or cvs.
